packages/cffi_example/meta.yaml

```diff
@@ -0,0 +1,11 @@
+package:
+  name: cffi_example
+  version: "0.1"
+requirements:
+  run:
+    - cffi
+source:
+  path: python-cffi-example
+test:
+  imports:
+    - cffi_example
```

Comment:
Could you add about/home link pointing to github as well and the license? Actually can't we install it from a Github release? Or did you make lots of changes to the example?

Reply:
I did not make many any changes to the example (just deleted some unneeded files). I don't think there is support in `pyodide-build` for loading a github repo, and they don't have seem to have made any releases. But I agree we should fix it to download it from github in some way.

Reply:
Even without a release you should be able to download it as https://github.com/wolever/python-cffi-example/archive/refs/heads/master.zip  Or with the appropriate commit in the URL. It's the "Download Zip" button,
![image](https://user-images.githubusercontent.com/630936/127750239-2f58afd1-b5bb-4b08-8ed4-e06c4f79bd23.png)